src/MSALWrapper/TokenFetcherPublicClient.cs

```diff
@@ -314,6 +314,8 @@ public async Task<TokenResult> GetTokenNormalFlowAsync(IPCAWrapper pcaWrapper, I
 
             try
             {
+                const string interactiveAuthMessage = "Opening a browser window for interactive auth...";

```

Comment:
We can use the warning message directly while logging the warning.

Reply:
Ah, the reason I did it this way was to avoid duplicating the same message string twice in the code.

Reply:
Makes sense. Probably move it as a `const string` on the top.

Reply:
Did you mean a `const string` at the class level, @shalinikhare27? If so, that's reasonable to me.

Reply:
That's right.